themes/doks/layouts/_default/list.html

```diff
@@ -21,7 +21,8 @@
 				</nav>
 			{{ end }}
 			{{ end }}
-			{{ if ne .Page.Params.version "Corda 5.0"}}			
+			{{ $c5 := in .Site.Params.corda5versions .Page.Params.version }}
```

Comment:
same comment (it's a boolean so usually in most programming languages it's indicated in the name ... e.g. `isCorda5Version`)

Reply:
Fix all occurrences please 😄

Reply:
https://r3-cev.atlassian.net/browse/DOC-5788